Command/CleanMediaCommand.php

```diff
@@ -25,11 +25,6 @@
 class CleanMediaCommand extends ContainerAwareCommand
 {
     /**
-     * @var MediaProviderInterface[]|false
-     */
-    private $providers = false;
```

Comment:
Yes. The commit message should let the reader understand : 

- what the issue was
- how you fixed it